examples/step-89/step-89.cc

```diff
@@ -508,6 +510,8 @@ namespace Step89
     template <typename VectorType>
     void evaluate(VectorType &dst, const VectorType &src) const
     {
+      // PM: we should consider to merge pressure_r_mortar and pressure_r
```

Comment:
Do you have an idea, how we could do it? Currently I can not think about a way to do it.

Reply:
Currently, the problem is the last template argument: `true` vs. `false`. Do we actually need this template argument?

Reply:
Yes. Dependent on this template argument, different types are chosen for `communicaton_objets` and different `reinit()` functions are enabled. This makes it possible to call `integrator->reinit(cell->index(),face_no)` in case of mortaring and `integrator->reinit(face)` for point-to-point interpolation.